packages/flutter/lib/src/material/material_state.dart

```diff
@@ -224,15 +222,15 @@ abstract class MaterialStateMouseCursor extends MouseCursor implements MaterialS
   @protected
   @override
   MouseCursorSession createSession(int device) {
-    return resolve(<MaterialState>{}).createSession(device);
+    return resolve(<MaterialState>{})!.createSession(device);
```

Comment:
@dkwingsmt Can you take a look at this? In general `resolve` will return a nullable type, I worry that this `!` isn't safe and should maybe look something more like:

```dart
return resolve(<MaterialState>{})?.createSession(device) ?? _defaultMouseCursor.createSession(device);
```

Reply:
Yes. In most (all) cases, if MaterialState's resolve method returns null, that indicates that the widget should defer to the next most general default.